cellarium/ml/models/scvi.py

```diff
@@ -17,6 +17,14 @@
     assert_columns_and_array_lengths_equal,
 )
 
+def weights_init(m):
```

Comment:
Is this function from somewhere else in the codebase?  If so, maybe we should consider refactoring so there is no copy-and-paste.

For now this is okay.